pkg/servicecentral/client.go

```diff
@@ -300,14 +300,8 @@ func (c *Client) GetService(ctx context.Context, user auth.OptionalUser, service
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to get attributes for service")
 	}
-	ogTeamAttr, found, err := findOpsGenieTeamServiceAttribute(resp)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to get OpsGenie attributes for service")
-	}
 
-	if found {
-		service.Attributes = append(service.Attributes, ogTeamAttr)
-	}
```

Comment:
Previously, we filtered the service attributes before adding them to the service object. We now do that filtering later on.